arbor/fvm_lowered_cell_impl.hpp

```diff
@@ -356,15 +356,17 @@ void fvm_lowered_cell_impl<B>::initialize(
     std::vector<cable_cell> cells;
     const std::size_t ncell = gids.size();
 
-    cells.reserve(ncell);
-    for (auto gid: gids) {
-        try {
-            cells.push_back(any_cast<cable_cell>(rec.get_cell_description(gid)));
-        }
-        catch (util::bad_any_cast&) {
-            throw bad_cell_description(rec.get_cell_kind(gid), gid);
-        }
-    }
+    cells.resize(ncell);
+    threading::parallel_for::apply(0, gids.size(), context_.thread_pool.get(),
+           [&](cell_size_type i) {
+               auto gid = gids[i];
+               try {
+                   cells[i] = std::move(any_cast<cable_cell>(rec.get_cell_description(gid)));
```

Comment:
The `std::move` here is redundant, but in order to remove one stage of move-construction, one can use `any_cast<cable_cell&&>`.

Reply:
Thanks! I hadn't looked closely at what `any_cast` was doing.